__tests__/api.test.js

```diff
@@ -217,8 +217,38 @@ describe('app', () => {
                   title: expect.any(String),
                   votes: expect.any(Number),
                   comment_count: expect.any(String)
-                }))
+                })
+              )
         })
       })
     })
+
+    describe("6. GET /api/reviews", () => {
+      test('should return an array of review objects sorted by date', () => {
+        return request(app)
+          .get('/api/reviews')
+          .expect(200)
+          .then(({ body }) => {
+            const { reviews } = body;
+            expect(reviews).toBeSortedBy('created_at', {descending: true});
+            expect(reviews).toBeInstanceOf(Array);
+            reviews.forEach((review) => {

```

Comment:
we should have an assertion for the length of the array before testing each of the elements